sections/browsers.include

```diff
@@ -59,28 +59,27 @@
   <a>creator browsing context</a>. Otherwise, the <a>browsing context</a> has no
   <a>creator browsing context</a>.
 
-  <p>If a <span>browsing context</span> <var>context</var> has a <span>creator browsing
-  context</span> <var>creator</var>, it also has the following properties. In what follows, let
-  <var>creator document</var> be <var>creator</var>'s <span>active document</span> at the time
+  <p>If a [=browsing context=] <var>context</var> has a [=creator browsing context=]
+  <var>creator</var>, it also has the following properties. In what follows, let
+  <var>creator document</var> be <var>creator</var>'s [=active document=] at the time
   <var>context</var> is created:</p>
 
   <dl>
    <dt><dfn>creator origin</dfn></dt>
-   <dd><var>creator document</var>'s <span>origin</span></dd>
+   <dd>|creator document|'s <a for="/">origin</a></dd>
 
    <dt><dfn>creator URL</dfn></dt>
-   <dd><var>creator document</var>'s <span data-x="concept-document-url">URL</span></dd>
+   <dd>|creator document|'s <a data-x="concept-document-url" for="/">URL</a></dd>
```

Comment:
data-x is not used by Bikeshed; please remove it.
for="/" will lead to the wrong link, rather you want to link to "concept-document-url" (see line ~425 of single-page.bs), which can be done by using the reference "document url" and displaying "URL" (line ~64 of browsers.include). So, long story short, use `[=document url|URL=]` for this.